core/runtime/src/main/java/io/quarkus/runtime/configuration/ConfigUtils.java

```diff
@@ -46,6 +47,12 @@
  *
  */
 public final class ConfigUtils {
+
+    /**
+     * The name of the property associated with a random UUID generated at launch time.
+     */
+    static final String UUID_KEY = "quarkus.uuid";
```

Comment:
I'm guessing that it is acceptable for a user to override the value, right? If that is the case, you need to add the property in `io.quarkus.runtime.ConfigConfig` so it gets included in the All Quarkus Configurations page.